README.md

```diff
@@ -26,7 +60,7 @@ For questions that are more suited for a forum we use the Qiskit tag in the
 
 ## Authors and Citation
 
-Qiskit Dynamics is the work of [many people](https://github.com/Qiskit/qiskit-terra/graphs/contributors) who contribute
+Qiskit is the work of [many people](https://github.com/Qiskit/qiskit-terra/graphs/contributors) who contribute
 to the project at different levels. If you use Qiskit, please cite as per the included [BibTeX file](https://github.com/Qiskit/qiskit/blob/master/Qiskit.bib).
```

Comment:
It would be good to make a separate citation / bib file specifically for this module rather than use the generic qiskit one.